apps/web-mzima-client/src/app/settings/surveys/create-field-modal/create-field-modal.component.scss

```diff
@@ -7,14 +7,53 @@
   justify-content: flex-start;
 }
 
+// HTML Button
+.list-item1 {
+  width: 220px;
+  border: none;
+  margin: 0 8px 36px 8px;
+  font-size: 14px;
+  line-height: 1.3;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  color: var(--color-neutral-90);
+  background: var(--color-neutral-10);
+  box-shadow: inset 0 0 0 1px var(--color-neutral-10);
+
+  &--disabled {
+    cursor: default;
+    color: var(--color-neutral-60);
+
+    h3 {
+      color: inherit;
+    }
+  }
+}
+
+.list-item1 h3 {
+  margin: 0 0 0.5rem 0;
+  color: var(--color-black);
+  font-size: 18px;
+  text-align: left;
+}
+
+.list-item1 p {
+  margin: 0;
+  text-align: left;
+}
+
+// Div clickable list items
 .list-item {
```

Comment:
What I don't understand is why you changed the padding, margin, and other styles for the original `list-item` class @kivuvarosekivuvan 

Let's restore this original class to its original state, since that's what's applied on the div and that's what we are comparing against. You therefore should adjust the duplicate new classes you create for the html `button` & `mzima-client-button` accordingly (where necessary).

Reply:
@Ifycode I will revert that to the initial padding and margin